webroot/src/components/Lists/ListContainer/ListContainer.ts

```diff
@@ -46,8 +46,18 @@ class ListContainer extends mixins(MixinListManipulation) {
         return size;
     }
 
-    get hasRecords() {
-        return (this.listData && this.listData.length);
+    get hasRecords(): boolean {
+        return Boolean(this.listData && this.listData.length);
```

Comment:
If this is a consistent style thing, then ignore me. But I don't recall us wrapping with the Boolean class in the past.

```suggestion
        return this.listData && this.listData.length;
```

Reply:
This will be a case where we likely should have Boolean casting. Since `hasRecords()` is contracted to return a boolean, it may not if we just rely on truthiness alone (e.g. if `this.listData` is truthy, then it would return that reference and not `true`).

Reply:
Ah yes. I missed that. Good point.